### PR TITLE
Cleanup EBCDIC string defintions

### PR DESCRIPTION
**Pull request by tmshort**
Use a single definiton for protocol string defintions.

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: ssl/statem/statem_lib.c
Comment (t8m) [inline ssl/statem/statem_lib.c:222]: Would it be possible to break these extra long lines?

Comment (tmshort) [reply ssl/statem/statem_lib.c:222]: Should be do-able.

**Review by hlandau — APPROVED** [review]
Good work, I found these duplicate definitions strange myself.

**Comment by slontis** [comment]
Stupid question: What is EBCDIC used for?

**Comment by paulidale** [comment]
IBM legacy.

**Comment by slontis** [comment]
> IBM legacy.

Is it still actually relevant? The wiki page for EBCDIC lists multiple jokes associated with the format..

**Comment by paulidale** [comment]
Sadly, yes it still is :(

**Comment by FdaSilvaYY** [comment]
Typos in commit messages : definition , 3 times

**Comment by t8m** [comment]
The krb5 external test is quite strange as it survives in re-runs and we do not see it anywhere else. @tmshort could you please try it locally? Or maybe just try to rebase this against up-to-date master branch although the test in CI should be running on a merged tree.

**Review by mattcaswell — APPROVED** [review]

**Comment by t8m** [comment]
Although this has the approvals I'd suggest not merging this until the krb5 test issue is resolved.

**Comment by slontis** [comment]
Is this hitting the string size limit somehow?
i.e. some of these were byte arrays before which can be large.

**Comment by tmshort** [comment]
> Is this hitting the string size limit somehow? i.e. some of these were byte arrays before which can be large.

The type hasn't changed. The strings aren't very big (less than 64 characters)

**Comment by tmshort** [comment]
> Although this has the approvals I'd suggest not merging this until the krb5 test issue is resolved.

I forced a re-run of the test, and it still failed. Will rebase when I can.

**Comment by tmshort** [comment]
Still failing after rebase. :(

**Comment by tmshort** [comment]
Found the issue, asking for re-approval.

**Review by mattcaswell — APPROVED** [review]

**Review by hlandau — APPROVED** [review]

**Comment by tmshort** [comment]
Merged to master!

**Comment by t8m** [comment]
@tmshort Please next time do not merge until the ready-to-merge label is set by the bot or 24 hours pass after the approval:done is set.

**Comment by tmshort** [comment]
Sorry @t8m, I had missed that!

**Pull request closed without merging**
